vllm/engine/arg_utils.py

```diff
@@ -297,14 +297,12 @@ def create_engine_configs(
     ) -> Tuple[ModelConfig, CacheConfig, ParallelConfig, SchedulerConfig,
                DeviceConfig, Optional[LoRAConfig]]:
         device_config = DeviceConfig(self.device)
-        model_config = ModelConfig(self.model, self.tokenizer,
-                                   self.tokenizer_mode, self.trust_remote_code,
-                                   self.download_dir, self.load_format,
-                                   self.dtype, self.seed, self.revision,
-                                   self.code_revision, self.tokenizer_revision,
-                                   self.max_model_len, self.quantization, 
-                                   self.sparsity, self.enforce_eager,
-                                   self.max_context_len_to_capture)
+        model_config = ModelConfig(
+            self.model, self.tokenizer, self.tokenizer_mode,
+            self.trust_remote_code, self.download_dir, self.load_format,
+            self.dtype, self.seed, self.revision, self.code_revision,
+            self.tokenizer_revision, self.max_model_len, self.quantization,
+            self.sparsity, self.enforce_eager, self.max_context_len_to_capture)
```

Comment:
looks like only a whitespace change here unless I'm missing something. Makes me concerned about the potential of merge conflicts due to changes like this. Are both we and the vllm folks using black?

Reply:
`yapf==0.32.0` should be used per the `requirements.txt`

Reply:
This was done automatically by `./format.sh`, Ill revert

Reply:
note: we fail the linting automation if I dont have this